test/src/e2e.long/invalidBlockPropagation.helper.ts

```diff
@@ -58,7 +58,6 @@ async function setup(): Promise<[Header, Block, Header]> {
         new H256(block0.transactionsRoot),
```

Comment:
Could you fix the commit message?

> We removed the unnecessary score.
The totalScore was being used in blockSyncMessage for the purpose of the seq, and we tried to
remove it. However, we need seq to check other peers have new blocks or
not. Therefore, we removed totalScore from blockSyncMessage and created
seq to use it.

Please write the current status. Foundry does not use `totalScore` in the `BlockSyncMessage`.
Our mockup test code was not updated when we change the `BlockSyncMessage`. (We should update the tests when we changed the BlockSyncMessage in the Foundry)
So, what you did is changing the mockup test code to follow Foundry's behavior.